lib/stack_master/version.rb

```diff
@@ -1,3 +1,3 @@
 module StackMaster
-  VERSION = "2.16.0"
+  VERSION = "2.17.8"
```

Comment:
A typo perhaps.

Reply:
I'll pop it back when folks are happy with this code, it just makes life easier for local testing (I can just do a bundle install to make sure I've got the right version)